hello-world/gradle.properties

```diff
@@ -0,0 +1,4 @@
+#
+#Thu Nov 04 16:23:52 GMT 2021
+conclaveVersion=1.2-RC3-SNAPSHOT
```

Comment:
Just remember to set this back to a "templated" version, once this work has completed. In the meantime, feel free to leave your custom version/repo here. Let's just keep this comment open, to remember (I know that there are "git" ways to avoid this, but they were not helpful to me, so I would proceed as suggested)